src/main/java/randoop/util/EquivalenceChecker.java

```diff
@@ -0,0 +1,30 @@
+package randoop.util;
+
+import randoop.types.PrimitiveTypes;
+
+/**
+ * Utility class for checking equivalence between objects in randoop.generation.Detective. Mainly
```

Comment:
I guess it is used by fuzzing as well as detective, or you wouldn't have included it in this pull request.

Reply:
Oops I forgot to remove it. It's not needed